test/sharness/t0081-repo-pinning.sh

```diff
@@ -279,6 +279,13 @@ test_expect_success "test add nopin dir" '
 
 '
 
+FICTIONAL_HASH="QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nm"
```

Comment:
maybe change it a little bit more so i don't second guess myself on whether or not thats the empty directory hash ;)